signature/jws/types.go

```diff
@@ -1,6 +1,36 @@
 package jws
 
-import "time"
+import (
+	"time"
+
+	"github.com/notaryproject/notation-core-go/signature"
+)
+
+const (
+	headerKeyExpiry                       = "io.cncf.notary.expiry"
+	headerKeySigningTime                  = "io.cncf.notary.signingTime"
+	headerKeyCrit                         = "crit"
+	headerKeyAlg                          = "alg"
+	headerKeyCty                          = "cty"
+	headerKeyAuthenticSigningTime         = "io.cncf.notary.authenticSigningTime"
+	headerKeySigningScheme                = "io.cncf.notary.signingScheme"
+	headerKeyVerificationPlugin           = "io.cncf.notary.verificationPlugin"
+	headerKeyVerificationPluginMinVersion = "io.cncf.notary.verificationPluginMinVersion"
+)
+
+func getHeaderKeys() []string {
```

Comment:
```suggestion
	headerKeys := []string{}
```
I prefer to use a variable instead because I don't see why we need a function. Are we going to add some logic to this function in the future?

Reply:
No logic here. Just for save memory. 
Changed it to be a variable.